src/main/java/frc/robot/subsystems/hood/Hood.java

```diff
@@ -49,6 +51,10 @@ public MutableMeasure<Angle> getAngle() {
         return inputs.angle;
     }
 
+    public MutableMeasure<Voltage> getPower() {
```

Comment:
This function should be called `getVoltage`.

Reply:
7b8b479